uvicorn/protocols/http/httpunk_impl.py

```
@@ -0,0 +1,489 @@
+                get_client_addr(www_scope),
+                scope["method"],
+                get_path_with_query_string(www_scope),
+                scope["http_version"],
+                status,
+            )
+
+    async def _send_500(self, request: Any, default_headers: list[tuple[bytes, bytes]]) -> None:
+        try:
+            await request.respond(
+                500,
+                headers=default_headers + [(b"content-type", b"text/plain; charset=utf-8")],
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)